frontend_lib/src/component/FilterBar/FilterBar.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import TextInput from '../Input/TextInput.jsx'
+
+require('./FilterBar.styl')
+
+const FilterBar = (props) => {
+  return (
+    <div className={`filterBar ${props.customClass}`}>
+      <TextInput
+        customClass='form-control'
+        {...props}
```

Comment:
Since props includes a customClass, it will overwrite the first one and class 'form-control' will never be sent to component TextInput, if props.customClass is only for component FilterBar to correct it, just invert the order (suggestion 1), otherwise you need to take both into account (suggestion 2)

```suggestion
        {...props}
        customClass='form-control'
```


```suggestion
        {...props}
        customClass={`form-control ${props.customClass}`}
```